redis/k8s-redis/nginx-controller.tf

```diff
@@ -0,0 +1,25 @@
+data "template_file" "ingresscontrollerconfig" {
```

Comment:
Lets maintain the camel case `ingress_controller_config` for the names that you are using.